app/src/main/java/com/duckduckgo/app/global/Url.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2017 DuckDuckGo
+ *
+ * Licensed under the   Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.duckduckgo.app.global
+
+import android.net.Uri
+import android.net.Uri.parse
+import com.duckduckgo.app.global.UrlScheme.Companion.http
+
+class UrlScheme {
```

Comment:
Nice. Although can i suggest we name the file to contain `Extension`. I've went for `ViewExtension`, `EditTextExtension` etc...

`UriExtension` perhaps?